src/components/DocumentationTopic/Hero/Hierarchy.vue

```diff
@@ -177,16 +173,24 @@ export default {
 @import 'docc-render/styles/_core.scss';
 
 .hierarchy {
```

Comment:
We may want to consider removing this for small width breakpoints or trying to better handle the limited space for this element in that scenario.

It looks kind of rough right now with long symbol names when there are multiple breadcrumbs:
<img width="425" alt="Screenshot 2024-01-29 at 1 35 27 PM" src="https://github.com/apple/swift-docc-render/assets/212918/810ca75e-0368-4d54-b1ff-11514e38d049">

Reply:
I've hidden the hierarchy in mobile viewport:
https://github.com/apple/swift-docc-render/pull/772/commits/65fe66c9c0caba23ab3b4a1fdd25bdc8a8d432dc